source/CScriptEngine.cpp

```diff
@@ -941,7 +941,22 @@ namespace CLEO
 
         GetInstance().ModuleSystem.LoadCleoModules();
         LoadState(GetInstance().saveSlot);
+
+        // keep already loaded scripts at front of processing queue

```

Comment:
why is it important to execute game scripts before custom ones? what does it solve?

Reply:
Handling save game pickup is done in main script. So any script that want to utilize **03D9
HAS_SAVE_GAME_FINISHED** needs to be executed after it. 
Due to multiple reload of scripts on game start it was already like that in CLEO4.